### ISSUE TITLE
Issue when running "make verilate"

### ISSUE THREAD

**Issue by Chentai-Y** [verilator/verilator#4539]
Hi Assignees,

When I run make verilate, I got error:

```
bender 0.27.3 available.
rm -rf build/verilator; mkdir -p build/verilator
./bender script verilator -t rtl -t ara_test -t cva6_test -t verilator --define NR_LANES=4 --define VLEN=4096 --define RVV_ARIANE=1 > build/verilator/bender_script_default
warning: Name issue with "cva6", `export_include_dirs` not handled
	Could relate to name mismatch, see `bender update`
/home/sy/source-code/ara/install/verilator/bin/verilator -f build/verilator/bender_script_default           \
  -GNrLanes=4                                                         \
  -O3                                                                           \
  -Wno-BLKANDNBLK                                                               \
  -Wno-CASEINCOMPLETE                                                           \
  -Wno-CMPCONST                                                                 \
  -Wno-LATCH                                                                    \
  -Wno-LITENDIAN                                                                \
  -Wno-UNOPTFLAT                                                                \
  -Wno-UNPACKED                                                                 \
  -Wno-UNSIGNED                                                                 \
  -Wno-WIDTH                                                                    \
  -Wno-WIDTHCONCAT                                                              \
  -Wno-ENUMVALUE                                                                \
  -Wno-COMBDLY \
  --hierarchical                                                                \
  tb/verilator/waiver.vlt                                                       \
  --Mdir build/verilator                                                       \
  -Itb/dpi                                                                      \
  --compiler clang                                                              \
  -CFLAGS "-DTOPLEVEL_NAME=ara_tb_verilator"                                        \
  -CFLAGS "-DNR_LANES=4"                                              \
  -CFLAGS -I/home/sy/source-code/ara/hardware/tb/verilator/lowrisc_dv_verilator_memutil_dpi/cpp       \
  -CFLAGS -I/home/sy/source-code/ara/hardware/tb/verilator/lowrisc_dv_verilator_memutil_verilator/cpp \
  -CFLAGS -I/home/sy/source-code/ara/hardware/tb/verilator/lowrisc_dv_verilator_simutil_verilator/cpp \
  ""                                                             \
  -LDFLAGS "-lelf"                                                              \
  ""                                                              \
  --exe                                                                         \
  /home/sy/source-code/ara/hardware/tb/verilator/lowrisc_dv_verilator_memutil_dpi/cpp/*.cc            \
  /home/sy/source-code/ara/hardware/tb/verilator/lowrisc_dv_verilator_memutil_verilator/cpp/*.cc      \
  /home/sy/source-code/ara/hardware/tb/verilator/lowrisc_dv_verilator_simutil_verilator/cpp/*.cc      \
  /home/sy/source-code/ara/hardware/tb/verilator/ara_tb.cpp                                           \
  --cc                                                                          \
                                       \
  --top-module ara_tb_verilator &&                                                  \
cd build/verilator && OBJCACHE='' make -j4 -f Vara_tb_verilator.mk
make[1]: Entering directory '/home/sy/source-code/ara/hardware/build/verilator'
make[2]: Entering directory '/home/sy/source-code/ara/hardware'
/home/sy/source-code/ara/install/verilator/bin/verilator -f build/verilator/Vlane_e_hierMkArgs.f
sh: 1: exec: /home/sy/source-code/ara/install/verilator/share/verilator/verilator_bin: not found
%Error: Command Failed ulimit -s unlimited 2>/dev/null; exec /home/sy/source-code/ara/install/verilator/share/verilator/verilator_bin -f build/verilator/Vlane_e_hierMkArgs.f
make[2]: *** [build/verilator/Vara_tb_verilator_hier.mk:361: hier_launch_verilator] Error 127
make[2]: Leaving directory '/home/sy/source-code/ara/hardware'
make[1]: *** [Vara_tb_verilator_hier.mk:370: Vlane_e/lane_e.sv] Error 2
make[1]: Leaving directory '/home/sy/source-code/ara/hardware/build/verilator'
%Error: make -C build/verilator -f Vara_tb_verilator_hier.mk  -j 1  hier_verilation exited with 2
%Error: Command Failed ulimit -s unlimited 2>/dev/null; exec /home/sy/source-code/ara/install/verilator/bin/verilator_bin -f build/verilator/bender_script_default -GNrLanes=4 -O3 -Wno-BLKANDNBLK -Wno-CASEINCOMPLETE -Wno-CMPCONST -Wno-LATCH -Wno-LITENDIAN -Wno-UNOPTFLAT -Wno-UNPACKED -Wno-UNSIGNED -Wno-WIDTH -Wno-WIDTHCONCAT -Wno-ENUMVALUE -Wno-COMBDLY --hierarchical tb/verilator/waiver.vlt --Mdir build/verilator -Itb/dpi --compiler clang -CFLAGS -DTOPLEVEL_NAME=ara_tb_verilator -CFLAGS -DNR_LANES=4 -CFLAGS -I/home/sy/source-code/ara/hardware/tb/verilator/lowrisc_dv_verilator_memutil_dpi/cpp -CFLAGS -I/home/sy/source-code/ara/hardware/tb/verilator/lowrisc_dv_verilator_memutil_verilator/cpp -CFLAGS -I/home/sy/source-code/ara/hardware/tb/verilator/lowrisc_dv_verilator_simutil_verilator/cpp  -LDFLAGS -lelf  --exe /home/sy/source-code/ara/hardware/tb/verilator/lowrisc_dv_verilator_memutil_dpi/cpp/dpi_memutil.cc /home/sy/source-code/ara/hardware/tb/verilator/lowrisc_dv_verilator_memutil_dpi/cpp/sv_scoped.cc /home/sy/source-code/ara/hardware/tb/verilator/lowrisc_dv_verilator_memutil_verilator/cpp/verilator_memutil.cc /home/sy/source-code/ara/hardware/tb/verilator/lowrisc_dv_verilator_simutil_verilator/cpp/verilated_toplevel.cc /home/sy/source-code/ara/hardware/tb/verilator/lowrisc_dv_verilator_simutil_verilator/cpp/verilator_sim_ctrl.cc /home/sy/source-code/ara/hardware/tb/verilator/ara_tb.cpp --cc --top-module ara_tb_verilator
make: *** [Makefile:168: build/verilator/Vara_tb_verilator] Error 2
```
And my verilator version is 5.012

Could you please take a look at the error? thanks!

Best,
Seven

**Comment by wsnyder** [comment]
>exec: /home/sy/source-code/ara/install/verilator/share/verilator/verilator_bin: not found

That file doesn't exist and needs to.  Something wasn't installed right, or you need to set VERILATOR_ROOT.
 
See the various run/install options here https://veripool.org/guide/latest/install.html#eventual-installation-options

Closing, but feel free to followup in this issue.

**Comment by Chentai-Y** [comment]
Hi @wsnyder 

Thanks for your reply. I have tried to set `VERILATOR_ROOT`. Here comes the issue:

I found my verilator_bin.exe was in `~/install/verilator/bin`, so I set the ROOT here.
However, it said cannot find `verilated_std.sv`, which is in the `~/install/verilator/share/verilator/include`

Thus, I got a bit confused: where should I set `VERILATOR_ROOT`? Should I set it to the toolchain source code place?

By the way, may I ask what is the basic time span for make verilate compiling? I set the `VERILATOR_ROOT` to my verilator source code place, and it takes very long time to run...

Thanks!
Seven
